src/main/java/com/sforce/ws/bind/TypeMapper.java

```diff
@@ -134,25 +151,27 @@
         return map;
     }
 
-    private String packagePrefix;
-    private String interfacePackagePrefix = null;
+    private final boolean javaTime;
+    private final String packagePrefix;
+    private final String interfacePackagePrefix;
+
+    public TypeMapper(String packagePrefix, String interfacePackagePrefix, boolean javaTime) {
```

Comment:
Please add a default constructor that maintains old behavior.

Reply:
✅